doc/src/modules/stats.rst

```diff
@@ -1,5 +1,72 @@
 Stats
 ===========
+SymPy statistics module
+
+Introduces a random variable type into the SymPy language.
```

Comment:
This is not required. It's already included using the line `.. automodule:: sympy.stats`